scripts/devcloud-test.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+# Copyright (C) 2023 Intel Corporation
+
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+workspace=$1
+compiler_c=$2
+compiler_cxx=$3
+build_type=$4
+
+set -e
+echo "Hostname: $(hostname)"
```

Comment:
isn't it redundant?

Reply:
I think it is useful to know on which hostname and compute-node the tests were run.